catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -204,4 +261,79 @@ public void setParser(Parser parser) {
         this.marshalConfigurator.addProperty(Marshaller.JAXB_FRAGMENT, true);
         this.parser = parser;
     }
+
+    public SourceMonitor registrySourceMonitor = new SourceMonitor() {
```

Comment:
Think this should be moved into the init method. We don't need to hang on to the reference do we?